### DISCUSSION TITLE
Can't submit application

### DISCUSSION THREAD

**Discussion by medusasss** [community/community#131174] [GitHub Education]
### Hi everyone,

Hi, I've been trying to submit my application but everytime I click on "submit" the page refreshes but nothing happens. I've tried using Chrome and Safari on Ios, changing wifi or using mobile connection, vpn, all without luck. What should I do?

**Comment by CodewithWitcher** [comment]
Here are some steps you can take to troubleshoot and resolve the issue:

1. **Clear Cache and Cookies**:
   - In Chrome, go to Settings > Privacy and Security > Clear browsing data.
   - In Safari, go to Preferences > Privacy > Manage Website Data, then click Remove All.

2. **Disable Browser Extensions**:
   - Disable any extensions that might be interfering with the submission process.

3. **Check for Errors**:
   - Open the Developer Tools (F12 or right-click > Inspect) and check the Console and Network tabs for any error messages when you try to submit.

4. **Try Incognito/Private Mode**:
   - Open a new incognito window in Chrome or a private window in Safari and try to submit again.

5. **Update Browsers**:
   - Ensure that you are using the latest version of your browsers.

6. **Check GitHub Status**:
   - Visit [GitHub Status](https://www.githubstatus.com/) to see if there are any ongoing issues with GitHub's services.

7. **Check Form Validations**:
   - Ensure all required fields are filled out correctly. There may be client-side validation preventing submission.

8. **Review Submission Requirements**:
   - Double-check that your application meets all the requirements and formats specified by GitHub.

9. **Contact GitHub Support**:
   - If none of the above steps work, reach out to [GitHub Support](https://support.github.com/contact) for assistance.

If you encounter any specific error messages or issues in the Developer Tools, please share those details.

**Reply by medusasss** [reply]
I've done all the steps on mobile since I don't have a webcam on my pc and the application won't let me upload instead of take a picture. I've done all the checks I could do on mobile but nothing worked. It also doesn't give me an error message, it just refreshes the page and I need to start all over with the application process.

**Comment by HamdiAl-Maweri** [comment]
GitHub is experiencing degraded performance and delays in their Actions runs and Webhooks.
Keep an eye on [[GitHub Status]](https://www.githubstatus.com/) for updates regarding their service recovery.

**Reply by medusasss** [reply]
Tried again more times since the status is "all systems operational" but it still doesn't work...

**Comment by rahul-muthu** [comment]
Same issue here, Any mods care to help?

**Reply by medusasss** [reply]
Honestly it’s bumming me a lot, I’ve tried for a week already and no one seems to care.

**Comment by cinghioGithub** [comment]
I'm having the same issue. I tried both mobile and desktop, with different browsers, but nothing worked.

**Reply by medusasss** [reply]
Hopefully they’ll fix it soon, but so far the replies have been totally unhelpful.
